inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -508,4 +508,17 @@ i.fa-chevron-right.facet-slider {
 
 .brief-results-sortby {
 	color: #737373;
+}
+
+div > .modal-dialog {
+    width: 900px;
+}
+
+#textarea-cite {
+    resize: none;
+    width: 868px;
+    min-height: 237px;
+    background-color: #f5f5f5;
```

Comment:
What is this color? Is it something we use in other places and we can add as a [variable](https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/modules/theme/static/scss/base/variables.scss) ?